js/theme/fonts.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Utility functions to manage font properties to style mapping for both iOS and Android
+ * Fonts are managed differently on Android and iOS. Read the Font section of the
+ * README file included in this repository.
+ *
+ * @flow
+ */
+
+import { Platform } from 'react-native'
+
+const font = Platform.OS === 'android' ? 'TitilliumWeb' : 'Titillium Web'
+
+const fontWeights = {
+  '300': 'Light',
+  '400': 'Regular',
+  '600': 'SemiBold',
+  '700': 'Bold'
+}
+
+export type FontWeight = $Keys<typeof fontWeights>
+
+export type FontStyleObject = {
+  fontFamily: string,
+  fontWeight?: string,
+  style?: 'normal' | 'italic'
+}
+
+/**
+ * Get the correct fontFamily name on both Android and iOS
+ */
+export const makeFontFamilyName = (
```

Comment:
so here you could have:

```js
export const makeFontFamilyName = (
  osSelect: PlatformSelectSpec<string, string>, 
  weight?: FontWeight = '400',
  isItalic?: boolean = false
): string => osSelect({
  android: `${font}-${fontWeights[weight]}${isItalic ? 'Italic' : ''}`,
  ios: font
})
```

Reply:
PlatformSelectSpec seem just an object, you can't call

```
osSelect({
  android: `${font}-${fontWeights[weight]}${isItalic ? 'Italic' : ''}`,
  ios: font
})
```

Am i wrong?

Reply:
yes sorry, just look at [this code](https://github.com/facebook/react-native/blob/26684cf3adf4094eb6c405d345a75bf8c7c0bf88/Libraries/Utilities/PlatformOS.ios.js#L20)

Reply:
I've updated my example